qcfractal/cli/qcfractal_server.py

```diff
@@ -453,6 +456,25 @@ def server_user(args, config):
         sys.exit(1)
 
 
+def server_dashboard(args, config):
+
+    print("QCFractal server dashboard.\n")
+
+    print(f"QCFractal server base folder: {config.base_folder}")
+
+    print("\n>>> Checking the PostgreSQL connection...")
+
+    # psql = PostgresHarness(config, quiet=False, logger=print)
+    # ensure_postgres_alive(psql)
+
```

Comment:
Should this be `try... except(ImportError): print("Dashboard not installed. Please install foo package from conda")`?